modules/cluster/outputs.tf

```diff
@@ -6,7 +6,7 @@ locals {
     vpc_id                = var.vpc_config.vpc_id
     private_subnet_ids    = var.vpc_config.private_subnet_ids
     node_security_group   = aws_eks_cluster.control_plane.vpc_config.0.cluster_security_group_id
-    node_instance_profile = var.iam_config.node_role
+    node_instance_profile = regex("^arn:aws:iam::\\d+:role/*.*/(.+)?", var.iam_config.node_role_arn)[0]
```

Comment:
perhaps this is not always true when we pass `iam_config.node_role_arn` without the IAM module. Let me consider more 🤔

Reply:
should be fine to update with a variable for [`asg_node_group`](https://github.com/cookpad/terraform-aws-eks/blob/7f79fd495912bb03243abd8af83c62b517b6af2a/modules/asg_node_group/variables.tf#L9-L9). Let me keep the change.